applications/launchpad/gui-react/src/containers/MiningContainer/MiningBox/index.tsx

```diff
@@ -249,13 +253,21 @@ const MiningBox = ({
 
   const content = componentForCurrentStatus()
 
+  let helpPromptClick
```

Comment:
it's better to use `const`, like `const x = helpMessages ? () => {...} : undefined`